vcr/credential/validator_test.go

```diff
@@ -30,7 +35,9 @@ import (
 )
 
 func TestNutsOrganizationCredentialValidator_Validate(t *testing.T) {
```

Comment:
I would have expected that those new funcs to have their own tests. Keeps the test small.